packages/awareness/src/global.css

```
@@ -77,111 +105,284 @@
+  max-width: 100%;
+  overflow-wrap: anywhere;
   @apply text-awareness-muted font-awareness text-[13px] leading-[1.4];
 }
 
 .awareness-live-cursor {
-  transition: transform 80ms linear;
-  @apply absolute top-0 left-0 z-20 inline-flex items-start pointer-events-none
-    will-change-transform;
+  --awareness-live-cursor-label-x: 0px;
+  --awareness-live-cursor-label-y: 3px;
+  font-size: 12px;
```

Stylelint (17.10.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)